azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -335,6 +335,12 @@ internal class CallingMiddlewareActionHandlerImpl(
                     }
                 }
 
+                if ((callInfoModel.callingStatus == CallingStatus.CONNECTED || callInfoModel.callingStatus == CallingStatus.IN_LOBBY) &&
+                    callInfoModel.callStateError == null
```

Comment:
instead of doing the same null check multiple times, can we just do:
```
if (callInfoModel.callStateError != null) {
   ...
} else {
   ...
}
```